test/unit/HoldConfirmation.test.js

```diff
@@ -4,7 +4,7 @@ import sinon from 'sinon';
 import { expect } from 'chai';
 import { mount } from 'enzyme';
 // Import the component that is going to be tested
-import HoldConfirmation from './../../src/app/components/HoldPage/HoldConfirmation.jsx';
```

Comment:
No, but I wanted to fix this for future tests since it breaks the coverage report.